google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/reflection/GenericParent.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Would you mind making these entity classes used in the `FieldIteratorTests` nested in the field iterator tests? Similar how you did it for the `SchemaUtilsTests`.

Reply:
Will do. I did that initially and the field iterator actually (correctly) finds extra references to `this$0` if they are defined on the same class. I'll add a comment though